src/style.css

```diff
@@ -1 +1,12 @@
 /* styles go here */
+.input {
+  margin-bottom: 10px;
+}
+
+.input:last-child {
+  margin-bottom: 0;
+}
+
+.form-field {
```

Comment:
Consider adding a more descriptive class name, such as 'form-field-wrapper' instead of 'form-field', to make it clear that this class is related to the form.